### ISSUE TITLE
TODO: add missing "SDL_JOYBATTERYUPDATED" event

### ISSUE THREAD

**Issue by furious-programming** [PascalGameDevelopment/SDL2-for-Pascal#80]
There is a new event called `SDL_JOYBATTERYUPDATED`, which provides information about a change in joystick battery status. Corrently undocumented, but exists in the sources (last):

```c
/* Joystick events */
SDL_JOYAXISMOTION  = 0x600, /**< Joystick axis motion */
SDL_JOYBALLMOTION,          /**< Joystick trackball motion */
SDL_JOYHATMOTION,           /**< Joystick hat position change */
SDL_JOYBUTTONDOWN,          /**< Joystick button pressed */
SDL_JOYBUTTONUP,            /**< Joystick button released */
SDL_JOYDEVICEADDED,         /**< A new joystick has been inserted into the system */
SDL_JOYDEVICEREMOVED,       /**< An opened joystick has been removed */
SDL_JOYBATTERYUPDATED,      /**< Joystick battery level change */
```
https://www.libsdl.org/tmp/SDL/include/SDL_events.h

It fits to keep an eye on it.

**Comment by Free-Pascal-meets-SDL-Website** [comment]
Thanks

**Comment by suve** [comment]
Fixed by PR #95.
